src/containers/user-profile/comments-with-rating-block/CommentsWithRatingBlock.tsx

```diff
@@ -44,10 +38,27 @@ const CommentsWithRatingBlock = ({
       ? 'userProfilePage.reviews.titleTutor'
       : 'userProfilePage.reviews.titleStudent'
 
-  const items: MockReview[] =
-    userRole === UserRoleEnum.Tutor
-      ? [...responseMock.items]
-      : [...responseMockStudents.items]
+  const getReviews = useCallback(
+    () => ReviewService.getUserReviews({ userId, userRole }),
+    [userId, userRole]
+  )
```

Comment:
if function doesn't fit into single line, you can wrap it with braces
```suggestion
  const getReviews = useCallback(() => {
     ReviewService.getUserReviews({ userId, userRole }),
   },
    [userId, userRole]
  )
```